python/py-klein/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
+
+name                py-klein
+version             17.10.0
+checksums           rmd160  b1bec823abe8b0749c04c06b6b458183f9f64460 \
+                    sha256  30aaf0d78a987d5dbfe0968a07367ad0c73e02823cc8eef4c54f80ab848370d0 \
+                    size    26736
+
+categories-append   devel
+platforms           darwin
+license             MIT
+maintainers         {ijackson @JacksonIsaac} openmaintainer
+supported_archs     noarch
+
+description         A micro-framework for developing production-ready web services with Python
+long_description    ${description}
+homepage            https://github.com/twisted/klein
+
+master_sites        pypi:k/${python.rootname}
+distname            ${python.rootname}-${version}
+
+python.versions     36 37
+
+if {${name} ne ${subport}} {
+    depends_lib-append  \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-twisted \
+        port:py${python.version}-idna \
+        port:py${python.version}-werkzeug
```

Comment:
It's minor, but please sort the dependencies.

Reply:
Thanks for pointing this out. I was actually wondering how the list of dependencies should be ordered.